test/system/src/test/resources/pravega_withTLS.properties

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+autoScale.mute.time.seconds=120
+autoScale.cooldown.time.seconds=120
+autoScale.cache.expiry.seconds=120
+autoScale.cache.cleanUp.interval.seconds=120
+curator-default-session-timeout=10000
+bookkeeper.ack.quorum.size=3
+controller.transaction.lease.count.max=120000
+controller.retention.frequency.minutes=2
+log.level=DEBUG
+hdfs.replaceDataNodesOnFailure.enable=false
+controller.security.auth.enable=true
+controller.security.pwdAuthHandler.accountsDb.location=/opt/pravega/conf/passwd
+controller.security.auth.delegationToken.signingKey.basis=secret
+autoscale.controller.connect.security.auth.enable=true
+autoscale.security.auth.token.signingKey.basis=secret
+#4GB of cache.
+pravegaservice.cache.size.max=4294967296
+#4GB of cache + a buffer for whatever else Netty might need it for.
+io.netty.maxDirectMemory=5368709120
+pravega.client.auth.token=YWRtaW46MTExMV9hYWFh
+pravega.client.auth.method=Basic
+# tls.crt 
+controller.security.tls.enable=true
+controller.security.tls.trustStore.location=/etc/secret-volume/tls.crt
+controller.security.tls.server.certificate.location=/etc/secret-volume/tls.crt
+controller.security.tls.server.privateKey.location=/etc/secret-volume/tls.key
+controller.security.tls.server.keyStore.location=/opt/pravega/conf/server.keystore.jks
+controller.security.tls.server.keyStore.pwd.location=/opt/pravega/conf/server.keystore.jks.passwd
+controller.rest.tlsKeyStoreFile=/opt/pravega/conf/server.keystore.jks
+controller.rest.tlsKeyStorePasswordFile=/opt/pravega/conf/server.keystore.jks.passwd
+pravegaservice.certFile=/etc/secret-volume/tls.crt
```

Comment:
Properties from lime 38 to 41 are also old properties. Replace them with the new properties. The mapping between old and new properties can be found here: https://github.com/pravega/pravega/wiki/Configuration-Name-Changes.

Reply:
@ravibeta config lines from lies 36 to 39 still use old names, but the comment is marked resolved. Can you please fix them?